src/Components/App/App.js

```diff
@@ -161,28 +214,32 @@ class App extends Component {
             <Result determineRandomActivity={this.determineRandomActivity}/>
           </Route> */}
           <Route
-            exact 
-            path='/:activity'
-            render={({match}) => {
-              return <BrowsePage
-                name={match.params.activity}
-                data={this.state[match.params.activity]}
-              />
-            }}>
-          </Route>
+            exact
+            path="/:activity"
```

Comment:
We could probably change all of these `exact path` for the Route so that they are on one line.